### ISSUE TITLE
GUI is not handling pagination

### ISSUE THREAD

**Issue by rascaraficci** [dojot/dojot#344]
The map visualization and device list pages are not handling pagination. They show at most 20 devices.

**Comment by mmagr** [comment]
During our tests today, I'd use an "invalid" url scheme:
```shell
curl -sS "${target}/device?idsOnly?page_size=30" -H "${token}" | python -m json.tool
```
Notice the two sets of queryParams being set: `?idsOnly` and `?page_size=30`
Using a "valid" url, the API works as expected:
```shell
curl -sS "${target}/device?page_size=30" -H "${token}" | python -m json.tool
```

Anyways, the PR referenced just cleans up the code a bit.

**Comment by mmagr** [comment]
This should have been fixed at the latest (0.2.0-nightly20180326) baseline.
@rascaraficci  can you please verify?
